File: Level-0/44.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i = 1; i <= n; i++){
            int cnt = 0;
            for(int j = 1; j <= n; j++){
                for(int k = 1; k <= n; k++){
                    if(j * k == i){
                        if(++cnt == 3){
                            answer++;
                            break;
                        }
                    }
                }
            }
        }
        return answer;
    }
}

File: Level-0/44.js
function solution(n) {
    let answer = 0;
    
    for(let i = 1; i <= n; i++) {
        let cnt = 0;
        for(let j = 1; j <= Math.sqrt(i); j++) {
            if(i % j === 0) {
                if(j === i / j) {
                    cnt += 1;
                }else {
                    cnt += 2;
                }
            }
        }
        
        if(cnt >= 3) answer++;
    }

    return answer;
}
